### ISSUE TITLE
Unable to use Flow with Parcel

### ISSUE THREAD

**Issue by alexeyraspopov** [parcel-bundler/parcel#1526]
# 🐛 bug report

PR https://github.com/parcel-bundler/parcel/pull/682 introduced dependency on `serialize-to-js` that has indirect dependency on [`config-chain`](https://github.com/dominictarr/config-chain) that loads `test` to the installable bundle. The folder includes purposely invalid JSON that is getting detected by Flow. The author refuses to prevent publishing `test` folder (https://github.com/dominictarr/config-chain/issues/24, https://github.com/dominictarr/config-chain/pull/25, https://github.com/dominictarr/config-chain/pull/31, https://github.com/dominictarr/config-chain/issues/34).

This is not particularly bug in Parcel. I'm doing my best to find the most suitable thread in Flow repo to continue the discussion. With this ticket I want to warn users and possibly discuss alternatives that don't depend on packages that are not maintained properly.

The workaround is to add `.*/node_modules/.*/test` to `[ignore]` section in `.flowconfig`.

## 🎛 Configuration (.babelrc, package.json, cli command)

.babelrc

```
{
  "presets": ["env", "react"]
}
```

package.json

```
{
  "devDependencies": {
    "babel-preset-env": "^1.7.0",
    "babel-preset-react": "^6.24.1",
    "flow-bin": "^0.74.0",
    "parcel-bundler": "^1.8.1"
  }
}
```

CLI command

```
yarn flow status
```

**Comment by DeMoorJasper** [comment]
Not sure if this issue should be in here?

It's probably better to comment or open an issue in flow or `serialize-to-js` repo

**Comment by alexeyraspopov** [comment]
Since https://github.com/dawsbot/config-chain/pull/31 was merged and `config-chain@1.1.12` was released, the issue described here is no longer reproducible. No upstream fixes needed since `config-chain -> js-beautify -> serialize-to-js -> parcel` uses safe patch-depending version ranges.

In case some of you still experience the issue, try reinstalling the dependencies in order to have `1.1.12` version of `config-chain` installed.
